### PR TITLE
Conference creation and general layout

### PR DESCRIPTION
**Pull request by nikkimartin**
Added in frontend for conference creation and hooked it up to backend.  Also fixed up some of the layout (dashboard & mobile).

### CODE REVIEW

**Comment by movingpylon** [comment]
The login error message is appearing behind the buttons now on the login page

File: app/Http/Controllers/ConferenceController.php [hunk 1]
```diff
@@ -37,8 +37,8 @@ private function validateConferenceJson($req) {
             "end" => "required|date_format:Y-m-d",
             "location" => "required",
             "description" => "string",
-            "hasTransportation" => "boolean|required",
-            "hasAccommodations" => "boolean|required"]);
+            "hasTransportation" => "boolean",
+            "hasAccommodations" => "boolean"]);
```
Comment (brandonson) [inline app/Http/Controllers/ConferenceController.php]: What happens if you don't pass in `hasTransportation` or `hasAccommodations`?  They're not nullable in the database and don't have defaults either, not sure why you've made them non-mandatory.

**Comment by brandonson** [comment]
Other than that I don't see any issues - I think ryan gets final call on this one though, I don't know enough about angular to say for sure.

File: app/Http/Controllers/ConferenceController.php (hunk 1 above)
Comment (nikkimartin) [inline app/Http/Controllers/ConferenceController.php]: not sure why I switched that. Changed it back.

**Comment by nikkimartin** [comment]
calendar in conference create has been fixed (was an issue w/ date format) and invalid credentials error should now show below fields and above buttons in login panel.

let me know if there's anything else or if i'm good to merge.

**Comment by movingpylon** [comment]
looks fine to me

**Comment by brandonson** [comment]
fine by me

**Pull request merged**